src/lightning_app/utilities/imports.py

```diff
@@ -17,6 +17,22 @@
 from typing import List, Union
 
 from lightning_utilities.core.imports import module_available
+from packaging.requirements import Marker, Requirement
+
+try:
+    from importlib import metadata
+except ImportError:
+    # Python < 3.8
+    import importlib_metadata as metadata  # type: ignore
+
+
+def _get_extras(extras: str) -> List[str]:
```

Comment:
Also maybe add a comment mentioning that this is used in the platform. So others don't think it's unused